src/components/FollowersFollowing.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from "react";
+import { Button } from "antd";
+import FollowersFollowingUsers from "./FollowersFollowingUsers";
+import { preFabProfiles } from "../test/testProfiles";
+
+const FollowersFollowing = (): JSX.Element => {
+  const [selectedListTitle, setSelectedListTitle] = useState<string>("");
```

Comment:
I might switch this to an enum instead of a string. There are only three states:

```typescript
enum ListStatus = {
  CLOSED;
  FOLLOWERS;
  FOLLOWING;
}
```

That way instead of string comparisons, we get type safety and completeness.